tests/test_level/test_level_01.py

```diff
@@ -481,10 +482,17 @@ def test_turn_left_nl(self):
 
     def test_turn_with_text_gives_error(self):
         code = "turn koekoek"
+        expected = "pass"
+
+        skipped_mappings = [
+            SkippedMapping(SourceRange(1, 1, 1, 13), hedy.exceptions.InvalidArgumentException)
+        ]
+
         self.single_level_tester(
             code=code,
-            extra_check_function=lambda c: c.exception.arguments['line_number'] == 1,
-            exception=hedy.exceptions.InvalidArgumentException)
+            expected=expected,
```

Comment:
I hesitated as well when I implemented this.

I first thought about separating it as we have now, that would make two tests, one where we check the still valid code, including the removed wrong code, and another where we check the exceptions.

But after writing one test I got confused because in the first test (when checking expected) the developer might overlook that there is a mistake in the Hedy code. This way it's more clear (I think) that the code is actually wrong and also where.